syllabus/syllabus-app/src/webapp/js/syllabus.js

```diff
@@ -314,7 +314,7 @@ function showConfirmDeleteAttachment(deleteButton, msgs, event){
 }
 
 function showConfirmDelete(deleteButton, msgs, event){
-	var title = $(deleteButton).parent().find(".editItemTitle").html();
+	var title = $(deleteButton).parent().parent().find(".syllabusItemTitle").html();
```

Comment:
This seems brittle and likely to break in the future if the hierarchy changes again. Wouldn't it make more sense to add an ID to the element rather than a class? With an ID, we can get the element regardless of where it is in the hierarchy, and this line will never break again in the future.

Reply:
Exactly, for me looks like the Syllabus main page was redesign but the JS functions were not updated, that's why this bug is happening in 19.x and 20.x. It can be implemented in different ways, I followed the pattern that was used fixing the hiearchy, my bad.